gtest/src/manager.rs

```diff
@@ -210,6 +214,15 @@ impl ExtManager {
                 .entry(message.destination())
                 .or_default()
                 .push(message.message().clone());
+            self.log.push(Message::new(
```

Comment:
log should be `Vec<StoredMessage>` due to latest changes

Reply:
So there won't be that huge constructor

Reply:
in master there is still log is still ```Vec<Message>``` 
waiting for the update)

Reply:
> in master log is still ```Vec<Message>``` 
> 
> waiting for the update)

Yeah, so I think this could be done by yourself in this PR ;)

That's really simple thing

Reply:
then the ```message``` field in ```Dispatch``` should be also changed to ```StoredMessage```? If so it will involve a lot of changes. I'm afraid that will be hard to merge